.travis.yml

```diff
@@ -0,0 +1,30 @@
+language: scala
+
+sudo: required
+
+scala:
+   - 2.11.12
+   - 2.12.5
```

Comment:
please us 2.12.4 as i just posted to development a few days ago that 2.12.5 is broken